.gitignore

```diff
@@ -33,3 +33,5 @@ templates/info/
 *\.egg-info/
 \.pytest_cache/
 .eggs/
+pytest.ini
```

Comment:
Not sure this should be excluded. Is there test config that should be shared for correct functionality?

Reply:
Hi Dave, this was a change that was carried over from another branch (when I was trying to get the `pytests` to work locally). I have to merge that branch as well. So unless you or @jsjiang want me to make a PR, I'm going to merge it directly to `develop` - since there are no code changes only a `READ_ME.md` file for setting up testing locally. Please let me know if I should make a PR for that. Thank you!

Reply:
ok, no problem Rushi - was mostly just curious.